### PR TITLE
Add Wake API for settings generator

### PR DESCRIPTION
**Pull request by nategraff-sifive**
Marking as a draft until [requirements.txt support is in api-languages-sifive](https://github.com/sifive/api-languages-sifive/pull/9).

### CODE REVIEW

File: wit-manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "commit": "a2e36e2c523c66d499810bd52a2956623e2066f6",
```
Comment (richardxia) [inline wit-manifest.json:3]: LGTM, and let me tag this commit of api-languages-sifive.git so that it's an "official" released version.

**Review by richardxia — APPROVED** [review]
LGTM!

File: wit-manifest.json (hunk 1 above)
Comment (richardxia) [reply wit-manifest.json:3]: This is now blessed as v0.2.0: https://github.com/sifive/api-languages-sifive/releases/tag/v0.2.0

Comment (nategraff-sifive) [reply wit-manifest.json:3]: Thanks!

**Pull request merged**